src/types.ts

```diff
@@ -73,6 +73,7 @@ export interface RadarTrackParams {
   metadata?: RadarMetadata;
   tripOptions?: RadarTripOptions;
   desiredAccuracy?: 'high' | 'medium' | 'low';
+  fraud?: boolean;
```

Comment:
Is this what's is called in mobile SDKs? This boolean acts as a flag, so a verb might be more appropriate (`useFraud`, `checkFraud`, `enableFraud` etc...)